recipes/tmtk/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "tmtk" %}
+{% set version = "0.4.4.dev3" %}
+{% set sha256 = "d94600bd60b120ae96ef9e6fad5019114836bf6da0bc181113ab7433a2d04f75" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
It looks like there is an `entry_point` for this package:

https://github.com/thehyve/tmtk/blob/master/setup.py#L86-L88

Please add it:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

Reply:
Done.